src/sdk.ts

```diff
@@ -105,15 +105,9 @@ export class FuseSDK {
     const initialFees = BigInt(txOptions.feePerGas);
     this.setWalletFees(initialFees);
 
-    if(isIndependentTransaction) {
+    if (isIndependentTransaction) {
```

Comment:
not sure I like this name because independentTransaction pretty vague.
maybe we can call this `newNonceSequence`

Reply:
@MusabShakeel576 cc

Reply:
I concur. A more suitable name would be `useNonceSequence`. 
WDYT @leonprou ?

Reply:
Sounds great!